core/src/test/scala/unit/kafka/server/SimpleFetchTest.scala

```diff
@@ -69,8 +69,10 @@ class SimpleFetchTest {
   @Before
   def setUp() {
     // create nice mock since we don't particularly care about zkclient calls
-    val zkUtils = EasyMock.createNiceMock(classOf[ZkUtils])
-    EasyMock.replay(zkUtils)
+    val zkClient = EasyMock.createNiceMock(classOf[ZkClient])
```

Comment:
This is left over from changes required when leveraging zookeeper to track deletes. I left it because it looks like its closer to what was actually trying to be mocked based on the comment. I can keep it or leave it.